IM_calculation/Advanced_IM/runlibs_2d.py

```diff
@@ -27,15 +27,15 @@ def parse_args():
     parser = argparse.ArgumentParser()
 
     parser.add_argument(
-        Components.c000.str_value,
+        f"comp_{Components.c000.str_value}",
```

Comment:
is there a reason behind this change? because doing so will likely break codes like 
https://github.com/ucgmsim/IM_calculation/blob/9092fc3ee869d220a053ad3882e95456cfbe4928/IM_calculation/Advanced_IM/runlibs_2d.py#L102

Reply:
run.py (not just SDI, see run.py@Taghavi_Miranda_2005) calls runlibs_2d.parse_args() and expects to get their arguments to be like args.comp_000, args.comp_090.

Reply:
If this is only specific to SDI and T&M 2005, I'm happy to revert the change to runlibs_2d and fix SDI and T&M run.py instead.

Reply:
@jasonmotha Your thoughts?

Reply:
the reason it was coded this way is to avoid local definition of Components, and use the ones from qcore.
so instead of a local
https://github.com/ucgmsim/IM_calculation/blob/5de0f138b4e971ca902dcf96beafc3a631ec7264/IM_calculation/Advanced_IM/Models/Burks_Baker_2013/run.py#L22
its using 
https://github.com/ucgmsim/IM_calculation/blob/9092fc3ee869d220a053ad3882e95456cfbe4928/IM_calculation/Advanced_IM/runlibs_2d.py#L78
and to extract the value from args using 
https://github.com/ucgmsim/IM_calculation/blob/9092fc3ee869d220a053ad3882e95456cfbe4928/IM_calculation/Advanced_IM/runlibs_2d.py#L102

Reply:
i would prefer a centralized string and enum. thats why it was coded that way

Reply:
OK - Fixed as per @shot0829 's suggestion. Also fixed T&M 2005.